Resources/Prototypes/Entities/Constructible/Power/power_base.yml

```diff
@@ -268,3 +268,83 @@
   - type: ClientEntitySpawner
     prototypes:
     - HVDummyWire
+
+- type: entity
+  id: SolarAssembly
+  name: solar assembly
```

Comment:
i'd advise using new localization for the name and description, check Resources/Prototypes/Entities/Objects/Tools/gas_tanks.yml and Resources/Locale/en-US/entities/gas-tanks.ftl for examples